tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/Matter/MatterRepository.cs

```diff
@@ -1073,6 +1073,7 @@ public bool SetItemPermission(ClientContext clientContext, IList<IList<string>>
 
                     if (null != roleCollection && 0 < roleCollection.Count && 0 < usersToRemove.Count)
                     {
+                        usersToRemove = usersToRemove.Where(user => user != string.Empty).ToList();
```

Comment:
@WilsonReddy From the client an empty record is getting added and to avoid an extra loop this linq statement has been added will check why an empty row is getting added to  usersToRemove list